zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1208,7 +1208,32 @@ property, when available, is noted below.
 
     The default value is false.
 
-
+* *serializeLastProcessedZxid.enabled*
+  (Jave system property: **zookeeper.serializeLastProcessedZxid.enabled**)
```

Comment:
No, I don't see anyone wants to turn it off unless we run into unexpected issue.

Reply:
Yeah, it probably doesn't hurt, but we should not implement something just because it doesn't hurt. Also don't need to prepare feature flags for unexcpeted issues. These are not valid reasons. 

Either way, you can leave it as it is. One more thing: the doc mentions that this option is required for the Admin's snapshot command to work properly. If that's the case the Admin command should check whether this option is enabled, but I don't see the check there, right?

Reply:
Yes, you are right. Added the check and also the unit test.